__tests__/routes/recipient.test.js

```diff
@@ -17,6 +17,7 @@ const testUser1 = {
   zip_code: "08259-2106",
   veteran_status: false,
   household_size: 8
+  // ""
```

Comment:
Not quite sure what this is? But looks like a comment you no longer need, maybe remove that.